/tmp/oauthenticator/setup.py:36:5: DUO105 use of "exec" is insecure
